### ISSUE TITLE
build fails with fontforge error

### ISSUE THREAD

**Issue by joaocc** [moinejf/abc2svg#8]
Hi,
We are having some difficulties in building the library.

```
$ ninja -v 
[1/12] fontforge -c 'open("font/abc2svg.sfd").generate("abc2svg.woff")'
[2/12] cp editor/edit.css edit-1.css
[3/12] if [ -x /usr/bin/git -a -d .git ]; then	VERSION=`git describe`;	VERSION=${VERSION#v};	DATE=`git log -1 --pretty=format:%ci HEAD | cut -d' ' -f1`;else	VERSION=`grep '"version"' package.json|cut -d'"' -f4`;	DATE=`stat --printf=%y build.ninja | cut -d' ' -f1`;fi;cp core/license abcdoc-1.js;echo "// abcdoc-1.js for abc2svg-$VERSION ($DATE)" >> abcdoc-1.js;if [ "x$NOMIN" = "x" ]; then	uglifyjs util/abcdoc.js >> abcdoc-1.js;else	cat util/abcdoc.js >> abcdoc-1.js;fi
[4/12] if [ -x /usr/bin/git -a -d .git ]; then	VERSION=`git describe`;	VERSION=${VERSION#v};	DATE=`git log -1 --pretty=format:%ci HEAD | cut -d' ' -f1`;else	VERSION=`grep '"version"' package.json|cut -d'"' -f4`;	DATE=`stat --printf=%y build.ninja | cut -d' ' -f1`;fi;cp core/license abcemb-1.js;echo "// abcemb-1.js for abc2svg-$VERSION ($DATE)" >> abcemb-1.js;if [ "x$NOMIN" = "x" ]; then	uglifyjs util/abcemb.js >> abcemb-1.js;else	cat util/abcemb.js >> abcemb-1.js;fi
FAILED: fontforge -c 'open("font/abc2svg.sfd").generate("abc2svg.woff")'
Copyright (c) 2000-2012 by George Williams.
 Executable based on sources from 14:57 GMT 31-Jul-2012-ML.
 Library based on sources from 14:57 GMT 31-Jul-2012.
Traceback (most recent call last):
  File "<string>", line 1, in <module>
AttributeError: 'file' object has no attribute 'generate'
ninja: build stopped: subcommand failed.
```

Can you please help identify what is the cause? 
Although this also happens on OSX (Windows doesn't even run for obvious reasons), we created a Dockerfile to have a clear and consistent environment (https://github.com/highskillz/abc2svg/blob/master/Dockerfile).

Thanks

**Comment by moinejf** [comment]
Your fontforge is a bit old (2012 - mine is 2016).
Anyway, I included the file 'font.js' in the repository in the last version, but there was still a dependency problem. This one is fixed by the commit f1363749733e5851591a9a8ab6d54955e8bd2f40.
So, you should be able to do the generation without fontforge.

**Comment by joaocc** [comment]
Upgrading fontforge solved it. Updated the Dockerfile. Thx.
